webui/react/src/pages/F_ExpList/glide-table/icons.tsx

```diff
@@ -0,0 +1,32 @@
+export const CompactHeight: React.FC = () => (
```

Comment:
should we prefer this to the practice of using `.svg` files in `assets/images`?

Reply:
I don't have major feelings either way, but `import`ing the files returns a path to the file to be used as a src, which doesn't allow the svg to use the `currentcolor` keyword.